tests/wpunit/Tribe/Tickets/Commerce/PayPal/Stock/SharedCapacityTest.php

```diff
@@ -0,0 +1,285 @@
+//
+//		$this->assertEquals( $remaining_b, $config->global_stock->get_stock_level() );
+//	}
+//
+//	/**
+//	 * @test
+//	 * @skip
+//	 */
+//	public function after_orders_global_stock_and_sales_count_should_match() {
+//
+//		$config = $this->get_config();
+//
+//		$ticket_a_sold = $config->ticket_a->qty_sold();
+//		$ticket_b_sold = $config->ticket_b->qty_sold();
+//
+//		$total_sold = $ticket_a_sold + $ticket_b_sold;
+//
+//		$this->assertEquals( $ticket_a_sold + $ticket_b_sold, $config->global_stock->tickets_sold() );
+//		$this->assertEquals( tribe_get_event_capacity( $config->event_id ) - $total_sold, $config->global_stock->get_stock_level() );
+//	}
+//
+//	/**
+//	 * @test
+//	 * @skip
+//	 */
+//	public function after_deleting_capped_ticket_attendee_the_available_count_should_increase() {
+//		$config = $this->get_config();
+//
+//		$ticket_a_attendees = $config->provider->get_attendees_by_id( $config->ticket_a->ID );
+//
+//		// Number of sold tickets should be the same as number of attendees.
+//		$this->assertEquals(  $config->ticket_a->qty_sold(), count( $ticket_a_attendees ) );
+//
+//		$first_attendee_id = $ticket_a_attendees[0]['attendee_id'];
+//
+//		$this->assertNotEmpty( $first_attendee_id );
+//
+//		$prev_data = [
+//			'inventory' => $config->ticket_a->inventory(),
+//			'available' => $config->ticket_a->available(),
+//			'capacity'  => $config->ticket_a->capacity(),
+//			'stock'     => $config->ticket_a->stock(),
+//		];
+//
+//		// Global stock before deletion.
+//		$prev_stock = $config->global_stock->get_stock_level();
+//
+//		// Deleting the first ticket.
+//		$config->provider->delete_ticket( $config->event_id, $first_attendee_id );
+//
+//		// refresh ticket.
+//		$config->ticket_a = $config->provider->get_ticket( $config->event_id, $config->ticket_a->ID );
+//
+//		// refresh attendee count.
+//		$ticket_a_attendees = $config->provider->get_attendees_by_id( $config->ticket_a->ID );
+//
+//		// Number of attendees should be 1 less then number of tickets sold.
+//		$this->assertEquals(  $config->ticket_a->qty_sold() - 1, count( $ticket_a_attendees ) );
+//
+//		// Ticket Stock data should be updated accordingly.
+//		$this->assertEquals( $prev_data['inventory'] + 1, $config->ticket_a->inventory() );
+//		$this->assertEquals( $prev_data['available'] + 1, $config->ticket_a->available() );
+//		$this->assertEquals( $config->shared_config['capped_limit'], $config->ticket_a->capacity() );
+//		$this->assertEquals( $prev_data['stock'] + 1, $config->ticket_a->stock() );
+//
+//		// Global Stock should be updated by 1. Commented out for now as we need to fix this in the main code.
+//		$this->assertEquals( $prev_stock + 1, $config->global_stock->get_stock_level() );
+//
+//		$ticket_a_product = wc_get_product( $config->ticket_a->ID );
+//
+//		// Ticket A product stock is synced with Ticket Object stock.
+//		$this->assertEquals( $config->ticket_a->stock(), $ticket_a_product->get_stock_quantity() );
+//	}
```

Comment:
Are we leaving this commented?

Reply:
@juanfra can we leave commented out code like this? I left this here on purpose like the above ones as these are the tests that need to be expanded when we get time in the future.

For now, those commented out tests are left out as the PayPal tester functions are not working as required and we might need to fix them to make this work as expected.